functional/keylime_create_policy-static-data/test.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+# vim: dict+=/usr/share/beakerlib/dictionary.vim cpt=.,w,b,u,t,i,k
+. /usr/share/beakerlib/beakerlib.sh || exit 1
+
+rlJournalStart
+
```

Comment:
Please, add here one `rlPhaseStartSetup` containing only `rlAssertRpm keylime` like we do have here
https://github.com/RedHat-SP-Security/keylime-tests/blob/main/functional/basic-attestation-on-localhost/test.sh#L14
The purpose is to check that keylime is installed and have its version logged.

Reply:
Fixed.